utilities/ggd/README.md

```diff
@@ -0,0 +1,48 @@
+ggd - GoGitDiff
+===============
+
+This is a tool to compute the packages affected by the difference between two Git
+revisions. It's primary goal is to help speed up CI jobs by figuring out the packages
```

Comment:
"It was deSigned to speed up C.I jobs by determining which packages depend on packages that were changed in the target branch vs. the base branch" or something like that.

Also would be cool to link to Digital Ocean's blog post on the topic

Reply:
Edit: nevermind you did link to it